tests/c/keep_includes.c

```diff
@@ -0,0 +1,23 @@
+//===--- keep_includes.c - test input file for iwyu -----------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// keep_includes-d1.h is an empty file.
+// Normally it would be removed.
+// In this test we are exercising the --keep command line option.
+// IWYU should not remove keep_include-d1.h when --keep=*keep_includes-d1.h
+// is used.
+
+#include "keep_includes-d1.h"
```

Comment:
Most tests use a path from the header search path, i.e. `tests/c/keep_includes-d1.h` -- does that work here, or does it confuse the globbing?